server/aws-lsp-codewhisperer/src/language-server/workspaceContext/artifactManager.ts

```diff
@@ -368,11 +368,19 @@ export class ArtifactManager {
             onlyFiles: true,
         })
 
+        const hasJavaFile = files.some(file => file.endsWith('.java'))
```

Comment:
in this function when you get files, please use fast glob streaming methods. See https://github.com/aws/language-servers/pull/2003

Reply:
Ack let me refactor this to streaming methods

Reply:
@leigaol  added the same glob.stream with early exist on MAX_FILES